src/main/java/ee/tuleva/onboarding/error/ErrorHandlingControllerAdvice.java

```diff
@@ -62,4 +66,20 @@ public ResponseEntity<Object> handleErrors(ErrorsResponseException exception) {
     log.error("{}", exception.toString());
     return new ResponseEntity<>(exception.getErrorsResponse(), BAD_REQUEST);
   }
+
+  @ExceptionHandler(ExpiredJwtException.class)
+  public ResponseEntity<Object> handleErrors(ExpiredJwtException exception) {
+    log.debug("ExpiredJwtException {}", exception.toString());
+    return new ResponseEntity<>(
+        JwtTokenUtil.getExpiredTokenErrorResponse(), HttpStatus.UNAUTHORIZED);
```

Comment:
I would hardcode the error code and message in this class, not in the `JwtTokenUtil` class

Reply:
The error is used in two different places, so for a clear definition, I extracted it